src/answers.sql

```diff
@@ -1,15 +1,119 @@
 -- Your answers here:
 -- 1
+SELECT 
+    c.name AS country_name, 
+    COUNT(s.id) AS total_states
+FROM 
+    states s
+JOIN 
+    countries c
```

Comment:
When you assign the table's name after FROM or JOIN statement I prefer to use the 3 letters to improve readability in the query. I won't comment the same on the following queries but you should solve it there as well.